hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/SnapshotMatchMode.java

```diff
@@ -1,31 +0,0 @@
-/*
- * Copyright (C) 2023 Hedera Hashgraph, LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
```

Comment:
Is this intentionally deleted?

Reply:
Yes, intentionally deleted; I moved this and `SnapshotMode` to `com.hedera.services.bdd.spec.utilops.records` package, but then accidentally re-created them here by merging `develop`. 😝

Reply:
Ahh ok 😄